L1Trigger/TrackerTFP/interface/TrackFindingProcessor.h

```diff
@@ -0,0 +1,88 @@
+#ifndef L1Trigger_TrackerTFP_TrackFindingProcessor_h
+#define L1Trigger_TrackerTFP_TrackFindingProcessor_h
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "L1Trigger/TrackTrigger/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+#include "L1Trigger/TrackerTFP/interface/TrackQuality.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTypes.h"
+
+#include <utility>
+#include <vector>
+#include <deque>
+
+namespace trackerTFP {
+
+  // Class to format final tfp output
```

Comment:
The name TrackFindingProcessor suggests that this class runs the entire TFP chain. Consider renaming it if it just does some reformatting at end of chain.

Reply:
Any progress on this?

Reply:
no